packages/fork-choice/src/protoArray/computeDeltas.ts

```diff
@@ -77,7 +82,7 @@ export function computeDeltas(
       }
       // We ignore the vote if it is not known in `indices .
       // We assume that it is outside of our tree (ie: pre-finalization) and therefore not interesting
-      const nextDeltaIndex = indices.get(nextRoot);
+      nextDeltaIndex = indices.get(nextRoot);
```

Comment:
Can you create a local cache to resolve root -> index in the context of this function? the 4,000,000 lookups end up adding up I think